.github/workflows/main.yml

```diff
@@ -0,0 +1,104 @@
+name: taisei-build-tests
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v1.*
```

Comment:
We also have `v1.*.x` branches (not tags) for staging stable releases before they are tagged.

Reply:
Added!